docs/client-options.md

```diff
@@ -118,6 +118,7 @@ third party clients.
 - [Certify](https://github.com/webprofusion/Certify) GUI (.NET, WinForms)
 - [oocx/acme.net](https://github.com/oocx/acme.net) (.NET)
 - [kelunik/acme-client](https://github.com/kelunik/acme-client) (PHP)
+- ZeroSSL: [Portable Perl version](https://zerossl.com/installation.html#portablePerl), [Portable Win32/Win64 binaries](https://github.com/do-know/Crypt-LE/releases).
```

Comment:
It seems like ZeroSSL is being listed in too many sections... Windows, Perl, Docker, Browser.... And sometimes its called "le.pl".... I think its reaching the point of confusion.

> Well, the reason behind mentioning Perl there is that it points to specific instructions in regards to using it as is on Windows with portable Perl 

How about:
```
- [ZeroSSL](https://zerossl.com/installation.html#portablePerl) (Perl)
- [ZeroSSL Win32/Win64](https://github.com/do-know/Crypt-LE/releases)
```

> though I'd probably kept it as "ZeroSSL Win32/Win64 portable", since they are portable :) 

What does that mean? "They are portable"

@do-know Are you a developer of ZeroSSL? Are your binaries "official"?